doc/source/install-on-macosx.rst

```diff
@@ -28,7 +28,7 @@ To build Ray, first install the following dependencies. We recommend using
   brew update
   brew install cmake pkg-config automake autoconf libtool boost openssl bison wget
 
-  pip install numpy funcsigs click colorama psutil redis flatbuffers cython --ignore-installed six
```

Comment:
It should be fine to remove these dependencies, since the `pip install -e .` should pull them in, but if people ignore the `pip install` and just run the `setup.py` then they may see problems. Not sure if I should just leave these dependencies in here.

Reply:
@richardliaw thoughts about this?

Reply:
er I guess this is a little late; but it would be nice to find out why `pip install .` doesn't work on your computer. We should follow up on that and can merge a fix.